cmd/monogen/testapp/wallet/metamask-integration.js

```diff
@@ -30,18 +30,37 @@ async function connectMetamask() {
             signer
         );
 
-        console.log('Connected to MetaMask and contract instantiated.');
-    } catch (error) {
-        console.error('Error connecting to MetaMask:', error);
-    }
+    await addL1ChainToMetaMask();
+    await switchToMonomerDevnet();
+    await runKeplrIntegration();
```

Comment:
If we're including the keplr integration logic in this file as well should we just move all of the related funcs over from the `keplr-integration.js` file and remove that file?

Alternatively, can we remove the following [block](https://github.com/polymerdao/monomer/blob/main/cmd/monogen/testapp/wallet/keplr-integration.js#L84-L85) at the end of the `keplr-integration.js` file since it no longer applies?
```
const enableKeplrButton = document.getElementById("enable-keplr");
enableKeplrButton.addEventListener('click', runKeplrIntegration);
```

Reply:
Do you mind if we merge this PR and do a proper refactor in a follow-up? We can tackle code organization and linting in a single PR I think.

Reply:
Yeah sure that sgtm